### PR TITLE
feat:  add status box component

### PR DESCRIPTION
**Pull request by JairJosafath**

### CODE REVIEW

**Comment by JairJosafath** [comment]
image

mobile view, I removed the borders

**Comment by JairJosafath** [comment]
image

image

File: packages/ui-components/src/components/status-box-component.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  render() {
+    return html`
+      <div class="flex justify-center items-center bg-neutral-800 p-6">
+        <div
+          class="w-11/12 md:flex md:justify-around md:max-w-4xl grid justify-center content-around md:h-full h-auto py-16"
+        >
+          ${this.boxes.map(
+            ({ value, label, color }) => html`
+              <div
+                class="flex flex-col items-center flex-nowrap md:border-[3px] border-b-2  border-neutral-600 md:hover:bg-neutral-700/25 md:hover:border-opacity-95 md:px-4 md:py-2 py-16 md:rounded sm:w-52 px-0 w-48 h-fit duration-150 hover:shadow-sm max-md:last:border-none "
```
Comment (jensk-dev) [inline packages/ui-components/src/components/status-box-component.ts]: change from md:border-[3px] to border-2 so it's consistent with other styling. Tailwind's classes are designed to implicitly create consistent styling across all elements. Introducing custom values between square brackets defeats the purpose of tailwind.

File: packages/ui-components/src/components/status-box-component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              >
+                <label
```
Comment (jensk-dev) [inline packages/ui-components/src/components/status-box-component.ts]: Labels must be used with an associated element. This is not semantically correct HTML, please use something else. Preferably a `span` or a smaller heading like `h3` or `h4`

File: packages/ui-components/src/exports/status-box-component.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { StatusBoxComponent } from '../components/status-box-component';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'status-box-element': StatusBoxComponent;
```
Comment (jensk-dev) [inline packages/ui-components/src/exports/status-box-component.ts:5]: rename to status-box-component to stay consistent with the class name

**Review by jensk-dev — CHANGES_REQUESTED** [review]
Small changes, feel free to ask if uncertain

**Review by xandervedder — APPROVED** [review]

**Pull request merged**